api/v2/controllers/bouquets/bouquets_controller.py

```diff
@@ -33,3 +33,23 @@ def post(self):
                                   refresh_token=credentials['refresh_token'])
         return make_response(jsonify({'data': {'message': 'successfully added bouquet',
                                                'bouquet': data}}), 201)
+    
+    def delete(self):
+        data = request.args['bouquet_id']
+
+        try:
+            #check the data if it is an integer
+            result = is_integer(data)
+            check_is_integer= isinstance(result, int)
+            if check_is_integer:
+                one_bouquet = query_one_bouquets(data)
+                if one_bouquet:
+                    BouquetsModel.delete_bouquet(BouquetsModel, data)
+                    return make_response(jsonify({'message': 'The bouquet was deleted successfully'}), 200)
```

Comment:
Nice work there pal, but I think the right status code is a 204 for resource deleted

Reply:
Thanks for the feedback. 200 is used on DELETE when the response contains an entity describing the status. With my code, I have that entity. 204 means No content and they can both be used.